README.md

```diff
@@ -36,23 +36,29 @@ The JWT may subsequently be used to proxy through to RAS services.
 
 Run the application
 -------------------
-Install dependencies for the application using pip
+Install pipenv
+```bash
+pip install pipenv
 ```
-pip install -r requirements.txt
+
+Use pipenv to create a virtualenv and install dependencies
+```bash
+pipenv install --dev
 ```
+
 Once these have been installed the app can be run from the ras-backstage directory using the following
 ```
 $ python run.py
```

Comment:
This might need changing to the application in the Pipenv-managed virtual environment: `pipenv run python run.py`